### ISSUE TITLE
Invalid token generated for template function

### ISSUE THREAD

**Issue by rafael-aero** [bridgedotnet/Bridge#3742]
After updating from 17.3 to 17.4, the following code generates an invalid token.

### Steps To Reproduce

Definition of the method: 
```csharp
public class ClassName
{
    public static async Task<MyClass<T>[]> FetchAsync<T>(MyClass<T>[] objects)
    {
        //code here
    }
}
```

being called as:
```csharp

var objects = new MyClass<HTMLSpanElement>[3];
// add a objects in the array
await ClassName.FetchAsync(objects);

```

### Expected Result

```js
$task1 = ClassName.FetchAsync(Bridge.global.HTMLSpanElement, objects);
```

### Actual Result

```js
$task1 = ClassName.FetchAsync(?, objects);
```

If I hard-code T = HTMLSpanElement, it generates the correct code.

**Comment by fabriciomurta** [comment]
Hello, @rafael-aero!

Thanks for reporting the issue! Are you able to reproduce this in a [deck](deck.net)?

**Comment by fabriciomurta** [comment]
As long as the deck below is accurate to your issue, it should be fixed as 17.5.0 is released:
https://deck.net/3f7b02c44454ab4ef3760933158479b5
